api/query_builder.go

```diff
@@ -141,15 +141,17 @@ func StatusCheckRollupGraphQL(after string) string {
 							...on StatusContext {
 								context,
 								state,
-								targetUrl
+								targetUrl,
+								isRequired(pullRequestNumber: $pr_number)
```

Comment:
I don't know a better way of solving this, but this might bite us in the future: this hardcodes a reference to a `$pr_number` variable that just so happens to be in the query that `StatusCheckRollupGraphQL` is going to be embedded in, but that's not guaranteed. It's possible that someone could change the parent query but forget the update the reference here, in which case we would get a failure in production (but likely not in tests).

Reply:
This currently breaks `pr status` for this exact reason.